lib/rules/disallow-capitalized-comments.js

```diff
@@ -57,7 +57,7 @@ module.exports.prototype = {
         var letterPattern = require('../../patterns/L');
         var lowerCasePattern = require('../../patterns/Ll');
 
-        file.getComments().forEach(function(comment) {
+        file.iterateTokensByType(['Line', 'Block'], function(comment) {
```

Comment:
It definitely feels weird to iterate by the types Line and Block and expect only comments. This should be renamed to LineComments and BlockComments but it appears that the code that is doing that is deeper than our project. At least, I couldn't find where those were being set in JSCS.